src/components/modal/ModalDialog.js

```diff
@@ -58,20 +60,25 @@ class ModalDialog extends React.Component<Props> {
      * @return {ReactElement|null} - Returns the button, or null if the button shouldn't be rendered
      */
     renderCloseButton() {
-        const { closeButtonProps, onRequestClose, intl } = this.props;
+        const { closeButtonProps, closeButtonText, onRequestClose, intl } = this.props;
         const { formatMessage } = intl;
         if (!onRequestClose) {
             return null;
         }
 
-        return (
+        const sharedProps = {
+            ...closeButtonProps,
+            'aria-label': formatMessage(messages.closeModalText),
```

Comment:
not sure if aria label is needed since button has text visible and screen reader should read it fine

Reply:
I guess you are using it for the X version, nvm